docs/Features.md

```diff
@@ -0,0 +1,108 @@
+# Application features
+
+## Connect
+
+You can connect to an IoT Central application by scanning a QR code in IoT Central.
+
+To learn more, see [Connect the app](#connect-the-app) later in this guide.
+
+## Telemetry
+
+The app collects data from sensors on the phone to send as telemetry to the IoT service you're using. Sensor data is aggregated every five seconds by default, but you can change this on the app settings page:
+
+![Telemetry](./media/telemetry.png)
+
+The following screenshot shows a device view in IoT Central that displays some of the device telemetry:
+
+![Telemetry](./media/central-telemetry.png)
+
+## Properties
+
+The app reports device status, such as device model and manufacturer. There's also an editable property that you can modify and see the change synchronize in your AzureIoT solution:
+
+![Properties](./media/properties.png)
+
+The following screenshot shows the writable property in IoT Central after the property was sent to the device:
+
+![Properties](./media/central-writable-property.png)
+
+## Image upload
+Both IoT Central and IoT Hub enable file upload to Azure storage from a device. The smartphone app lets you upload an image from the device.
+
+To learn more about configuring your service to support file uploads from a device, see:
+
+[Upload files from your device to the cloud with IoT Hub.](https://learn.microsoft.com/en-us/azure/iot-hub/iot-hub-csharp-csharp-file-upload)
+
+[Upload files from your device to the cloud with IoT Central.](https://learn.microsoft.com/en-us/azure/iot-central/core/howto-configure-file-uploads)
+
+![Upload](./media/image-upload.png)
+
+## Logs
+The smartphone app writes events to a local log file that you can view from within the app. Use the log file to troubleshoot and better understand what the app is doing:
+
+![Logs](./media/logs.png)
+
+## Settings
+The settings page in the app lets you:
+
+- Connect the app to your Azure IoT solution.
+- Review the current device registration information.
+- Reset the app by clearing the stored data.
+- Customize the app appearance.
+- Set the frequency that the app sends telemetry to your IoT service.
+
+![Settings](./media/settings.png)
+
+## Connect the app
```

Comment:
The steps from here onward feel like they should be somewhere else, maybe in the main README or a `Getting connected` doc. I feel like they could be missed in this file

Reply:
there is a link in main README to point to this. It would have been too noisy for a frontdoor readme